### PR TITLE
[MANX-274][dotnet] Remove unused code from MergedDateTimeParser

### PR DESCRIPTION
**Pull request by Bill7zz**
- Remove unused if statement from Japanese `MergeDateTimeParser` class

### CODE REVIEW

**Review by gasper-az — APPROVED** [review]
LGTM

File: .NET/Microsoft.Recognizers.Text.DateTime/Models/DateTimeModel.cs [hunk 1]
```diff
@@ -32,33 +32,24 @@ public List<ModelResult> Parse(string query, DateObject refTime)
...
+                {
+                    parsedDateTimes.Add(parseResults);
                 }
-
-                // Filter out ambiguous cases. Naïve approach.
-                parsedDateTimes = Parser.FilterResults(query, parsedDateTimes);
-            }
-            catch (Exception)
-            {
-                // Nothing to do. Exceptions in parse should not break users of recognizers.
```
Comment (JuanAr) [inline .NET/Microsoft.Recognizers.Text.DateTime/Models/DateTimeModel.cs]: I'm a bit concerned about removing a try catch IF the Parser can throw an exception.

Comment (JuanAr) [reply .NET/Microsoft.Recognizers.Text.DateTime/Models/DateTimeModel.cs]: I think, the PR will be approved maintaining the try/catch structure, removing the unused code if any.
Add a new task to review this specific smelling code: _Try/Catch in code without any explicit action_
Also, add a new issue in our repo to review. And then create the issue in MS repo to socialize and receive feedback and or approval to tackle it.

**Review by JuanAr — CHANGES_REQUESTED** [review]
Feedback regarding some functionality changes

**Review by JuanAr — APPROVED** [review]
👍

**Comment by JuanAr** [comment]
Closing because the class was refactored and this is no longer needed.

**Pull request closed without merging**